README.md

```diff
@@ -1,5 +1,5 @@
-<div align="center">
-<img alt="Coreax logo" src="https://raw.githubusercontent.com/gchq/coreax/main/documentation/assets/Logo.svg">
+<div style="align: center;">
+    <img alt="Coreax logo" src="https://raw.githubusercontent.com/gchq/coreax/main/documentation/assets/Logo.svg">
 </div>
```

Comment:
While the "align" tag is deprecated, it is the only way that I'm aware of, to perform alignment in GitHub readmes. This is the result of sanitization that occurs behind the scenes. In the gaffer [README](https://github.com/gchq/Gaffer/blob/7f8832f1ed2ddd43286434fc19689d0654c9bb91/README.md?plain=1#L1) the align tag is used similarly to here. 

Alternatively, we could just make the logo smaller and have it left aligned?

Reply:
Ah right, fair enough! I will revert this back to what we had before as we know that it works!